lms/static/sass/elements/_system-feedback.scss

```diff
@@ -145,6 +145,13 @@
   }
 }
 
+.wrapper-msg {
```

Comment:
It wasn't clear to me what the goal of having this abstraction here for the fixed variant of the wrapper. Should the .wrapper-msg-fixed class also be including other styles to set the fixed behavior?